app/forms/search_form.rb

```diff
@@ -82,7 +82,9 @@ def at_least_one_field_set
       send(field).present?
     end
 
-    errors.add(:base, :nothing_specified) unless field_set
+    search_type = { crm7: 'claim', crm4: 'application' }
```

Comment:
Nit: `'claim'` and `'application'` are technically user-facing strings, so shouldn't be hard-coded into our models. (If we were to offer our service in Welsh as well as English we'd have to refactor this). Really we should do something like:

```ruby
errors.add(:base, :nothing_specified, value: I18n.t("shared.submission_nouns.#{application_type}")) unless field_set
```

And then add something to config/locales/en.yml or similar